core/src/main/java/hudson/lifecycle/UnixLifecycle.java

```diff
@@ -42,11 +43,14 @@
  *
  * <p>
  * Restart by exec to self.
+ * The restartCommand configuration may override executbale name.
```

Comment:
typo "executbale". Probably it should be documented differently, because the current statement does not explain what is a `restartCommand`